concepts/55 Vue Components/20 Component Configuration Syntax/33 Two-Way Property Binding.md

```diff
@@ -18,4 +18,15 @@ Changes in the `bindingProperty` are propagated to the TextBox's **value** and v
             }
         }
     }
+    </script>
+
+    <!-- tab: App.vue (Composition API) -->
+    <template>
+        <DxTextBox v-model:value="bindingProperty" />
+    </template>
+
+    <script setup>
+    import DxTextBox from 'devextreme-vue/text-box';
+
+    let bindingProperty = 'Some value';
```

Comment:
I think it would be nice if we also add a note about Vue 2, like

[important] In Vue 2 applications, two-way binding should be implemented using the `sync` modifier. 
This small note will close the following cases:
https://supportcenter.devexpress.com/internal/ticket/details/T1134019
https://supportcenter.devexpress.com/internal/ticket/details/T1064927

Reply:
Ok, I will add such a note!

Reply:
it seems it's wrong two-way binding example. 
It should be: 

`import { ref } from 'vue';
let bindingProperty = ref('Some value');`